test/e2e/e2e_test.go

```diff
@@ -116,7 +124,52 @@ var _ = Describe("controller", Ordered, func() {
 				return nil
 			}
 			EventuallyWithOffset(1, verifyControllerUp, time.Minute, time.Second).Should(Succeed())
+		})
+
+		It("should delete the session when idle and hibernate hits threshold", func() {
+			session_namespace := "default"
+			projectDir, _ := utils.GetProjectDir()
+
+			By("creating an instance of the session")
+			EventuallyWithOffset(1, func() error {
+				cmd := exec.Command("kubectl", "apply", "-f", filepath.Join(projectDir,
+					"config/samples/culling-configured.yaml"), "-n", session_namespace)
+				_, err := utils.Run(cmd)
+				return err
+			}, time.Minute, time.Second).Should(Succeed())
```

Comment:
question: I cannot find this in the project files. There is no `cullilng-configured.yaml` in `config/samples` in the project.

Reply:
I forgot to add it ! Hence the test failure...

Fixed